curator-test-zk35/pom.xml

```diff
@@ -166,6 +166,12 @@
             <artifactId>slf4j-log4j12</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is this change necessary?

Reply:
https://github.com/apache/curator/actions/runs/3071902247/jobs/4962990393

Reply:
I see, sorry that I missed that :+1: